vhdbuilder/packer/install-dependencies.sh

```diff
@@ -21,7 +21,7 @@ echo "Components downloaded in this VHD build (some of the below components migh
 
 # fix grub issue with cvm by reinstalling before other deps
 # other VHDs use grub-pc, not grub-efi
-if [[ "${UBUNTU_RELEASE}" == "20.04" ]]; then
+if [[ "${UBUNTU_RELEASE}" == "20.04" ]] && [[ "$IMG_SKU" != "20_04-lts-cvm" ]]; then
```

Comment:
```suggestion
if [[ "${UBUNTU_RELEASE}" == "20.04" ]] && [[ "$IMG_SKU"=!= "20_04-lts-cvm" ]]; then
```

you have this backwards if you're trying to handle fips

Reply:
whoops typo. fixed.